cookbook/Makefile

```diff
@@ -20,18 +21,18 @@ in_container_register_sandbox:
 	pyflyte -p ${PROJECT} -d development --config /root/sandbox.config register workflows
 
 .PHONY: register_sandbox
-register_sandbox: docker_push
+register_sandbox: docker_build
```

Comment:
should we add a push and register?

Reply:
push and register?  nah.  pushing is only for those with push perms right?  That's a relatively small set of people and occasions I think.